src/launch/launch.scss

```diff
@@ -46,6 +46,7 @@
       height: 100%;
       display: grid;
       grid-template-rows: 2fr 1fr;
+      max-width: 600px;
```

Comment:
I am not sure why the cards have different widths (45%, 48%) but, you'll probably want to set the max-width on the surrounding divs rather than here, this way the cards will stay centered on wider screens.
Maybe 600px for the first div and 620px (≈+3%) for the second one?

Reply:
@giladt you want me to modify this ?
<img width="303" alt="Capture d’écran 2021-11-23 à 12 23 22" src="https://user-images.githubusercontent.com/11537417/143016044-a83eec6b-3746-4c0b-b2c4-5e47f0667e9f.png">

But the result is not good
<img width="603" alt="Capture d’écran 2021-11-23 à 12 25 48" src="https://user-images.githubusercontent.com/11537417/143016180-e338e16b-eb45-49d2-a931-26b4ebfaa0da.png">

Reply:
@giladt but when both have max-width : 600px 
<img width="560" alt="Capture d’écran 2021-11-23 à 12 27 00" src="https://user-images.githubusercontent.com/11537417/143016360-c6a9abfe-3833-4e45-b903-cab592bd5806.png">
it seems ok

Reply:
You're right. As said, I'm not sure why there's a width difference between the cards (maybe @dkent600 remembers the reason?). I think they can both have 45% and max of 600px. Also maybe a min-width is needed to avoid this (but not on smaller screens!):

![image](https://user-images.githubusercontent.com/2517870/143092600-8f5c0306-5174-400c-986b-61d1cfc429fc.png)

Reply:
I don't recall any reason for different widths.

Reply:
There are still some issues with the way the buttons wrap within the cards.
Under `launch-box-container` class, please apply the following CSS:
1. On `buttons` class- please add a column-gap of 10px.
2. On the `a` element decleration within the `buttons` element- please 
   a. set the left-margin to 0 and
   b. set a width of 100px.

Doing this should set the width of both buttons to be identical, align them to the left, and set a minimal gap before the `request information` button wraps.

Please test your work in all screen media queries to see if it function correctly.
Thanks!